### ISSUE TITLE
Medications, screen readers, Page titles do not reflect page contents.

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#64585]
### Point of contact

Angela Fowler

### Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Details

<h2>User story: </h2>
As a screen reader user, I expect the page title to reflect the contents of the page. Screen reader users often use Insert+T to read the page title and quickly determine where they are on a web page. For each individual prescription, the title is MHV Medications | Veterans Affairs. 
<h2>Explanation</h2>
The page title is the text in the page head between the "<title>" and "</title>" tags. To a screen reader user it renders at the very top of the page though it may be hidden from sighted users as the h1 serves the same purpose for them.
<h2>Recommendation</h2>
The title should match the h1. So in the case of ACETAMINOPHEN 160MG/5ML ALC-F LIQUID, both the title and h1 should be "ACETAMINOPHEN 160MG/5ML ALC-F LIQUID." Extra information such as the date last filled can also be provided. 
...

### Reproduction steps

<h2>User story: </h2>
As a screen reader user, I expect the page title to reflect the contents of the page. Screen reader users often use Insert+T to read the page title and quickly determine where they are on a web page. For each individual prescription, the title is MHV Medications | Veterans Affairs. 
<h2>Explanation</h2>
The page title is the text in the page head between the "<title>" and "</title>" tags. To a screen reader user it renders at the very top of the page though it may be hidden from sighted users as the h1 serves the same purpose for them.
<h2>Recommendation</h2>
The title should match the h1. So in the case of ACETAMINOPHEN 160MG/5ML ALC-F LIQUID, both the title and h1 should be "ACETAMINOPHEN 160MG/5ML ALC-F LIQUID." Extra information such as the date last filled can also be provided. 

### Proposed solution or next steps

...

### References, articles, or WCAG support

1.
2.
3.
...

### Type of issue

- [ ] Axe-core
- [X] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [ ] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else
